playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
```

Comment:
Move to test branch.

Reply:
Please delete. 
Our PRs need to be clean from unrelated changes, otherwise they won't be accepted by the company.
All our code will have to be merged into the official Elementor branch.